response_operations_ui/views/collection_exercise.py

```diff
@@ -100,12 +100,19 @@ def view_collection_exercise(short_name, period):
     info_panel = request.args.get('info_panel')
     sorted_nudge_list = get_existing_sorted_nudge_events(ce_details['events'])
 
+    # This is an ugly fix for errors being written to the permanently to the session. This guarantees that an error will
+    # be displayed once and then removed. If errors are ever tidied up (using flash for instance) then this code can go.
+    error_json = None
```

Comment:
nice fix but worth extracting to it's own function?

Reply:
Done